lib/bootstrap.dart

```diff
@@ -34,6 +36,7 @@ Future<void> bootstrap(
   await configureEncryptedStorage();
   await configureNavigationService();
   await configureNekoton();
+  await MigrationService.migrateWithHiveInit(inject<KeyValueStorageService>());
```

Comment:
Lets move any bootstrap-related things into bootstrap/{thing}.dart and call here something like 'migrateWithHive()' please.

Reply:
-